superset/views/sql_lab.py

```diff
@@ -176,13 +177,14 @@ def show(self, pk):
 appbuilder.add_view_no_menu(SavedQueryView)
 
 
-class TabStateView(BaseSupersetView):
-    def _get_owner_id(self, tab_state_id):
-        return db.session.query(TabState.user_id).filter_by(id=tab_state_id).scalar()
+def _get_owner_id(tab_state_id):
+    return db.session.query(TabState.user_id).filter_by(id=tab_state_id).scalar()
+
 
+class TabStateView(BaseSupersetView):
     @has_access_api
     @expose("/", methods=["POST"])
-    def post(self):
+    def post(self):  # pylint: disable=no-self-use
```

Comment:
That rule is pretty annoying. I wouldn't mind turning it off.

Reply:
I think it's valuable outside of this context. As I was doing this work, I identified quite a few methods that could be refactored out of a large class into a stand-alone function. I'd like to keep the rule for those instances.